.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
haha, you don't like my color theme for this repo

Reply:
Haha I didn't hate it, I just assumed it had been committed by mistake :p